orders/models.py

```diff
@@ -0,0 +1,21 @@
+from django.db import models
+
+from products.models import Products, Sizes, Grainding
+from users.models    import User
+from core.models     import TimeStampModel
+
+class Orders(TimeStampModel): 
+    order_number = models.CharField(max_length=250)
+    username     = models.ForeignKey(User, on_delete       = models.CASCADE)
+    status       = models.ForeignKey('Statuses', on_delete = models.CASCADE)
+
+class OrderItems(models.Model): 
+    product  = models.ForeignKey(Products, on_delete  = models.CASCADE)
+    size     = models.ForeignKey(Sizes, on_delete     = models.CASCADE)
+    graind   = models.ForeignKey(Grainding, on_delete = models.CASCADE)
+    order    = models.ForeignKey('Orders', on_delete  = models.CASCADE)
+    quantity = models.IntegerField()
+
+
+class Statuses(models.Model): 
+    status = models.CharField(max_length=150)
```

Comment:
```suggestion
    status = models.CharField(max_length=20)
```
결제 혹은 배송 관련 상태라고 한다면 예를 들어 `결제 대기중`, `결제 완료`, `출하 대기중`, `배송 중`, `배송 완료`, `구매 확정`등이 들어가게 될텐데 max_legnth를 150으로 설정한 것이 적절한지에 대한 고민이 필요할 수 있겠네요.

Reply:
수정했습니다.